nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -69,20 +63,22 @@ private static void LOG(boolean cond, String tag, String msg) {
   private final Map<String, Map<String, Map<Integer, Set<String>>>> argAnnotCache;
 
   private final Config config;
+  private final StubxCacheUtil cacheUtil;
```

Comment:
As a high level comment, I'm a bit unsure why we still need this handler even after we added the new library models handler.  Is it because of some Android-specific logic?  We should clean this up, but maybe not for this PR.